modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -12,16 +12,18 @@ def initialize(a_server, a_ttl)
     def do_create(name, value, type)
       zone = match_zone(name, enum_zones)
       msg = "Added #{type} entry #{name} => #{value}"
+      keyword_args = {msg: msg, error_only: false}
       value = "#{value}." if type == "PTR"
-      cmd = "/RecordAdd #{zone} #{name}. #{type} #{value}"
-      execute(cmd, msg)
+      cmd = ["/RecordAdd", zone, "#{name}.", type, value]
+      execute(*cmd, keyword_args)
```

Comment:
You can write this as:

    execute("/RecordAdd", zone, "#{name}.", type, value, msg: msg, error_only: false)

That is a tad cleaner :-)